minimal.rb

```diff
@@ -1,66 +1,54 @@
 run "if uname | grep -q 'Darwin'; then pgrep spring | xargs kill -9; fi"
 
-# GEMFILE
+# Gemfile
 ########################################
-inject_into_file 'Gemfile', before: 'group :development, :test do' do
+inject_into_file "Gemfile", before: "group :development, :test do" do
   <<~RUBY
-    gem 'autoprefixer-rails', '10.2.5'
-    gem 'font-awesome-sass'
-    gem 'simple_form', github: 'heartcombo/simple_form'
+    gem "autoprefixer-rails"
+    gem "font-awesome-sass", "~> 6.1"
+    gem "simple_form", github: "heartcombo/simple_form"
 
   RUBY
 end
 
-inject_into_file 'Gemfile', after: 'group :development, :test do' do
-  <<-RUBY
-
-  gem 'pry-byebug'
-  gem 'pry-rails'
-  gem 'dotenv-rails'
+inject_into_file "Gemfile", after: 'gem "debug", platforms: %i[ mri mingw x64_mingw ]' do
+  <<~RUBY
+    gem "dotenv-rails"
+    gem "pry-byebug"
+    gem "pry-rails"
```

Comment:
@dmilon, just ran the minimal template and I'm getting a Gemfile error due to the development/test group that looks like this and is missing some spaces:

```ruby
group :development, :test do
  # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
  gem "debug", platforms: %i[ mri mingw x64_mingw ]gem "dotenv-rails"
gem "pry-byebug"
gem "pry-rails"

end
```

So I think we should change add back the first return line:
```suggestion
inject_into_file "Gemfile", after: 'gem "debug", platforms: %i[ mri mingw x64_mingw ]' do
  <<~RUBY
  
    gem "dotenv-rails"
    gem "pry-byebug"
    gem "pry-rails"
```

Reply:
Fixed in https://github.com/lewagon/rails-templates/pull/141/commits/a0236571f805f6be84b7ed243a81bf7b7adba918 :heavy_check_mark: